docs/index.md

```diff
@@ -46,9 +46,12 @@ cd LibreLingo
 
 ### Web app
 
+You will need [Node](https://nodejs.org/en/). Note that this project is not yet compatible with Node v16.
+
 Install dependencies:
 
 ```sh
+yarn set version classic
```

Comment:
is this necessary to do? 🤔

Reply:
On my machine, yes. Maybe you can check if this reproduces for you:

```
~/src/LibreLingo$ yarn set version berry
➤ YN0000: Downloading https://github.com/yarnpkg/berry/raw/master/packages/yarnpkg-cli/bin/yarn.js
➤ YN0000: Saving the new release in .yarn/releases/yarn-2.4.2.cjs
➤ YN0000: Done in 1s 395ms

~/src/LibreLingo$ yarn web dev
/home/zeb/src/LibreLingo/.pnp.js:29908
  return Object.defineProperties(new Error(message), {
                                 ^

Error: Your application tried to access svelte-preprocess, but it isn't declared in your dependencies; this makes the require call ambiguous and unsound.

Required package: svelte-preprocess (via "svelte-preprocess")
Required by: /home/zeb/src/LibreLingo/apps/web/

Require stack:
- /home/zeb/src/LibreLingo/apps/web/webpack.config.js
- /home/zeb/src/LibreLingo/.yarn/$$virtual/sapper-virtual-474baa3db7/0/cache/sapper-npm-0.29.1-7c84c0fa20-214bf74633.zip/node_modules/sapper/dist/create_manifest_data.js
- /home/zeb/src/LibreLingo/.yarn/$$virtual/sapper-virtual-474baa3db7/0/cache/sapper-npm-0.29.1-7c84c0fa20-214bf74633.zip/node_modules/sapper/dist/dev.js
- /home/zeb/src/LibreLingo/.yarn/$$virtual/sapper-virtual-474baa3db7/0/cache/sapper-npm-0.29.1-7c84c0fa20-214bf74633.zip/node_modules/sapper/dist/cli.js
- /home/zeb/src/LibreLingo/.yarn/$$virtual/sapper-virtual-474baa3db7/0/cache/sapper-npm-0.29.1-7c84c0fa20-214bf74633.zip/node_modules/sapper/sapper
    at internalTools_makeError (/home/zeb/src/LibreLingo/.pnp.js:29908:34)
    at resolveToUnqualified (/home/zeb/src/LibreLingo/.pnp.js:30867:23)
    at resolveRequest (/home/zeb/src/LibreLingo/.pnp.js:30965:29)
    at Object.resolveRequest (/home/zeb/src/LibreLingo/.pnp.js:31043:26)
    at Function.external_module_.Module._resolveFilename (/home/zeb/src/LibreLingo/.pnp.js:30141:34)
    at Function.external_module_.Module._load (/home/zeb/src/LibreLingo/.pnp.js:30006:48)
    at Module.require (node:internal/modules/cjs/loader:996:19)
    at require (node:internal/modules/cjs/helpers:92:18)
    at Object.<anonymous> (/home/zeb/src/LibreLingo/apps/web/webpack.config.js:6:28)
    at Module._compile (node:internal/modules/cjs/loader:1092:14)
```